src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,67 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book("Principles of Programming", new BigDecimal("100.00"));
+        System.out.printf("New book created: %s",
+                bookDao.create(book));
+        System.out.println("\n\nLet's try to get our book by ID...");
+        long id = book.getId();
```

Comment:
Ensure that the `Book` class has a proper implementation for the `getId()` method. If the `id` is not set during the creation of the `Book` object, this may lead to unexpected behavior. Verify that the `id` is correctly assigned, possibly by the `BookDao` when the book is created.